packages/frontend/src/hooks/echarts/useEcharts.ts

```diff
@@ -640,12 +738,14 @@ const getEchartAxis = ({
                       })
                     : xAxisConfiguration?.[0]?.name ||
                       (xAxisItem ? getItemLabel(xAxisItem) : undefined),
-                min: validCartesianConfig.layout.flipAxes
+                min: !validCartesianConfig.layout.flipAxes
```

Comment:
I think this was a typo, on the bottom axis we should apply the values if it is not flipped

Reply:
min/max seems broken in your PR.

demo:
![Screenshot 2023-01-11 at 14 38 10](https://user-images.githubusercontent.com/9117144/211834473-82637150-5048-4e54-b254-347c312df827.png)

PR env:
![Screenshot 2023-01-11 at 14 39 09](https://user-images.githubusercontent.com/9117144/211834457-2a1fb7ac-e953-4fae-8865-95ab05a28a45.png)

Reply:
Yep, you're right, it was not a typo. It was there to apply the limit when flipped. I fixed it